Source/Admin/Web.Angular/src/app/project/add-project/add-project.component.ts

```diff
@@ -48,12 +56,19 @@ export class AddProjectComponent implements OnInit {
         );
     }
 
+    getSurveillanceOptionId(id: string){
```

Comment:
Todo: remove this after we change the template implementation according to comment above.